test/util/all_close.hpp

```diff
@@ -56,13 +56,14 @@ namespace ngraph
                     if (count < 5)
                     {
                         ar_fail << std::setprecision(std::numeric_limits<long double>::digits10 + 1)
-                                << a[i] << " is not close to " << b[i] << " at index " << i << "\n";
+                                << +a[i] << " is not close to " << +b[i] << " at index " << i
+                                << std::endl;
```

Comment:
Use of `\n` is often intentional because `std::endl` causes flushing, which can impact performance.
Why do you feel we need `std::endl` here?

Reply:
Performance in reporting test failures?

Reply:
The reason for `endl` is to support output on platforms like Windows, which don't use `\n` as the endline character.

I don't think we need to worry about performance in reporting test failures too much.